pkg/query-service/rules/thresholdRule.go

```diff
@@ -823,10 +823,11 @@ func (r *ThresholdRule) prepareLinksToLogs(ts time.Time, lbls labels.Labels) str
 		return ""
 	}
 
+	q := r.prepareQueryRange(ts)
 	// Logs list view expects time in milliseconds
 	tr := timeRange{
-		Start:    ts.Add(-time.Duration(r.evalWindow)).UnixMilli(),
-		End:      ts.UnixMilli(),
+		Start:    q.Start,
```

Comment:
do we need to convert to nanoSeconds as logs work in nanoSeconds precision?

Reply:
The logs list view expects them in millis.